tools/netsource.c

```diff
@@ -36,6 +36,7 @@ Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 
 #ifdef WIN32
 #include <winsock2.h>
+#define socklen_t int
```

Comment:
ouch was this one here before? it is nasty to define types like that.

Reply:
Has apparently been there since 2010 and is introduced by 02c76244.

Reply:
It is also still around in jack2: https://github.com/jackaudio/jack2/blob/a50aeb1f331546e86c1f35f26e67692c1cdfc9a5/tools/netsource.c#L43

Reply:
Do you want to apply a patch on top of this or add a ticket for it?

Reply:
lets leave it as-is then, but it is very bad practice